tests/e2e-scenarios/Readme.md

```diff
@@ -0,0 +1,28 @@
+# E2E SCENARIOS
+
+Contains E2E tests.
+
+Steps to run 'e2e-scenarios-suite':
+
+1) Follow the instructions from the [Readme](../../docs/Installation.md) scripts to install AppStudio in e2e mode
+2) Run the e2e suite: `./bin/e2e-appstudio --ginkgo.focus="e2e-scenarios-suite"`
```

Comment:
should [this script](https://github.com/redhat-appstudio/application-service/blob/86571e38f52e6bbe67b945dd58b695af26ae834d/.ci/oci-e2e-has.sh#L13) that's used by openshift-ci be updated too?

Reply:
yes, I think so.